firebase-firestore/src/androidTest/java/com/google/firebase/firestore/QueryTest.java

```diff
@@ -364,6 +364,75 @@ public void testCanListenForQueryMetadataChanges() {
     listener2.remove();
   }
 
+  /*
+  // TODO(orquery): Re-enable this test once the `where` API is public.
```

Comment:
Can we find a way to include this test in a non-commented out fashion. Ideas include:

- `@Ignore`
- Making the test call into a package-private method, preferably directly but if that is not possible via something like https://cs.opensource.google/firebase-sdk/firebase-android-sdk/+/master:firebase-firestore/src/androidTest/java/com/google/firebase/firestore/AccessHelper.java?q=accesshelper&ss=firebase-sdk

Commented code usually ages poorly.

Reply:
I realized I could remove the access modifier (rather than using `protected`) to enable testing and keeping the API bot happy.